docs/class-parser.md

```diff
@@ -0,0 +1,34 @@
+Class: Parser
+=============
+
+Parser is the class that will translate strings (starting with yamal) into properties (json) between storage and services
+
+## Interface
+
+### Class Attribute: `properties`
+
+```
+Parser.properties
+```
+
+The unmodified properties that need to be translated from human readable to json for services to read.
+
+### Class Attribute: `sources`
+
+```
+Parser.sources
+```
+
+The source of the properties. ie S3, Conqueso, ect
+
+### Constructor(properties (buffer), plugin_type (string))
```

Comment:
I was under the same impression John was. 
However, after the meeting yesterday, this documentation will be updated to reflect the new model.